bigtop-packages/src/rpm/ambari/SPECS/ambari.spec

```diff
@@ -391,7 +395,10 @@ if [ "$1" -eq 0 ]; then  # Action is uninstall
     if [ -d "/etc/ambari-agent/conf.save" ]; then
         mv /etc/ambari-agent/conf.save /etc/ambari-agent/conf_$(date '+%d_%m_%y_%H_%M').save
     fi
-    mv /etc/ambari-agent/conf /etc/ambari-agent/conf.save
+
+    if [ -d "/etc/ambari-agent/conf" ]; then
+        cp -r /etc/ambari-agent/conf /etc/ambari-agent/conf.save
```

Comment:
`cp -rp` should be better if there are configuration files having permissions like 600?

Reply:
Updated, thanks.